Sources/JWTProvider/Provider.swift

```diff
@@ -4,108 +4,45 @@ import JWT
 /// Adds required JWT objects to your application
 /// like token Signers
 public final class Provider: Vapor.Provider {
+
     public static let repositoryName = "jwt-provider"
     
-    public let signer: Signer
```

Comment:
Looks like this method and the init under it were removed w/o a deprecation, unless I missed it.